templates/csi-daemonset.yaml

```diff
@@ -55,7 +55,13 @@ spec:
           imagePullPolicy: {{ .Values.csi.image.pullPolicy }}
           args:
             - --endpoint=/provider/vault.sock
-            - --debug={{ .Values.csi.debug }}
+            {{- if .Values.csi.logLevel }}
+            - --log-level={{ .Values.csi.logLevel }}
+            {{- else if .Values.csi.debug }}
```

Comment:
Since `.csi.debug` existed prior to this change, we will probably want to honor the previous setting of `csi.debug`, otherwise we would be introducing a behavioral change that the user is not expecting.

Reply:
Very true! Good catch! This would be a breaking change, I reverted it. Thank you 🙏